armory/baseline_models/pytorch/sincnet.py

```diff
@@ -138,7 +136,7 @@ def preprocessing_fn(batch):
 
 # NOTE: PyTorchClassifier expects numpy input, not torch.Tensor input
 def get_art_model(model_kwargs, wrapper_kwargs, model_weights=None):
-    model = sincnet(**model_kwargs)
+    model = sincnet(model_weights=model_weights, **model_kwargs)
```

Comment:
So we unpack the `model_kwargs` but it would error. Is it safer for sincnet to accept kwargs? Or preferable to fail?

Reply:
If model_kwargs is an empty dict, it will work fine. If you add kwargs into it, it will fail. I would prefer it to fail over silently accepting unknown kwargs.